src/test/scala/edu/ie3/simona/agent/em/EmAgentIT.scala

```diff
@@ -285,18 +285,20 @@ class EmAgentIT
             emResult.getInputModel shouldBe emInput.getUuid
             emResult.getTime shouldBe 0L.toDateTime
             emResult.getP should equalWithTolerance(
-              -0.00034885012.asMegaWatt
+              -0.00057340027059.asMegaWatt
+            )
+            emResult.getQ should equalWithTolerance(
+              0.0000882855367033.asMegaVar
             )
-            emResult.getQ should equalWithTolerance(0.00008828554.asMegaVar)
         }
 
         scheduler.expectMessage(Completion(emAgentActivation, Some(7200)))
 
         /* TICK 7200
          LOAD: 0.269 kW (unchanged)
-         PV:  -3.651 kW
-         STORAGE: SOC 63.3 %
-         -> charge with 3.382 kW
+         PV:  -3.791 kW
+         STORAGE: SOC 63.3 % fixme: reminder check this at the end
```

Comment:
SoC is still at 63.3 % => 5000 W * 0.95 (eta of converter) = 4750 W, 2h of operation => 9500 Wh / 15000 Wh capacity = 63,3%